datasets/pk/proscribed_persons/crawler.py

```diff
@@ -0,0 +1,33 @@
+from zavod import Context
+from zavod import helpers as h
+
+
+def crawl_person(context: Context, row: dict):
+    person_name = row.pop("name")
+    father_name = row.pop("fatherName")
+    cnic = row.pop("cnic")
+    province = row.pop("province")
+    district = row.pop("district")
+
+    entity = context.make("Person")
+    entity.id = context.make_slug(person_name, prefix="pk-proscribed")
+    entity.add("idNumber", cnic)
+    entity.add("name", person_name)
+    entity.add("fatherName", father_name)
+    entity.add("topics", "crime.terror")
```

Comment:
let's do sanction topic and sanction entity with `4th Schedule under the Anti Terrorism Act, 1997` as sanction program.